test/specs/core.scale.tests.js

```diff
@@ -495,6 +495,9 @@ describe('Core.scale', function() {
 		it('should default to one layer for custom scales', function() {
 			var customScale = Chart.Scale.extend({
 				draw: function() {},
+				buildTicks: function() {
```

Comment:
Does something break without this?

Reply:
Yes. In `core.scale` I assume that `buildTicks` exists and builds the ticks. I though that was probably a safe assumption for any real code, but it required to update this test code to be more realistic

Reply:
It's a public API and if it worked without altering `this.ticks` then it's not safe to change that behavior.

Reply:
Ok, I've reverted the test changes